capstone.css

```diff
@@ -445,89 +445,89 @@ footer {
 
 @media only screen and (max-width: 768px) {
   .Hamburger {
-  display: inline-block;
-  height: 20px;
-  width: 20px;
+    display: inline-block;
+    height: 20px;
+    width: 20px;
   }
 
   .firsthead {
-  padding-bottom: 0;
+    padding-bottom: 0;
   }
 
   .firsthead h1 {
-  font-size: 30px;
+    font-size: 30px;
   }
 
   .program {
-  height: 930px;
+    height: 930px;
```

Comment:
- [ ] Please try to work on the padding and margin properties of your cards here in the `main program` section so as to match the design given 😎 

**Your design**
![image](https://user-images.githubusercontent.com/90222110/191797674-598f71bc-f2ac-414d-92ac-2e4dcc471300.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/191797848-30ed2034-7eb3-44c2-b307-6de05f71042f.png)

- [ ] Also remember to add that encircled link in the given design screenshot 😉

Reply:
Ok. I will add that. 
Thank you for your vivid review which helps me to enrich my knowledge regarding design.

Reply:
I did it.

Reply:
I made the active bar accordingly.